schema.graphql

```diff
@@ -345,7 +345,7 @@ type Mutation {
   deleteDiscount(id: IdInputField!): DeleteDiscountMutation
   discountRequirement(input: DiscountRequirementMutationInput!): DiscountRequirementMutationPayload
   deleteDiscountRequirement(id: IdInputField!): DeleteDiscountRequirementMutation
-  register(email: String!, firstName: String!, lastName: String!, password1: String!, password2: String!): Register
+  register(turnstileToken: String, email: String!, firstName: String!, lastName: String!, password1: String!, password2: String!): RegisterTurnstile
```

Comment:
Technically, this should be `turnstileToken: String!`, as it's a required (read: non-null) argument.

Reply:
Ah yes - looks like [here](https://github.com/BristolSTA/uobtheatre-web/pull/641/files#diff-204ad7265ed9910b0b2654a7d9faccfe7bd432c8d4b4dc35c1534b972cb1f7cdR7) in your code changes, we are defining it as non-null.